docs/intro/quickstart.md

```diff
@@ -0,0 +1,83 @@
+
+## **What you'll learn**
+> - Creating layouts (add widgets)
+> - Adding interactivity to UI elements
+> - Handle app behavior in response to user interactions (manage state).
+> - Running your app
+
+<br/>
+The steps to build the app are as follows:
+
+1. [Clone or create project](#clone-project)
+2. [Building UI](#build-ui)
+3. [Customize style](#customize-style)
+4. [Add state](#add-state)
+5. [Run the app](#run-app)
+
+## 1. Clone or create project {#clone-project}
+
+To kick off your project, the first step is to [create a new project](#). However, to make things easier, we've already created a starter app for you. Simply open [this link](https://app.flutterflow.io/project/f-f-quick-start-app-umu392), click the '**Clone**' button, and the project will be instantly added to your account. 
+
+<img src="../../static/img/clone-project.png" alt="clone-project.png"  />
+
+After cloning the project, you’ll see a page with product images and descriptions. You’ll add a feature that allows users to update the product quantity.
+
+<img src="../../static/img/zero-to-final.png" alt="zero-to-final.png"  />
+
+## 2. Building UI {#build-ui}
+
+In this section, we'll see how to build the user interface (UI) for this feature. This involves creating a layout and adding various widgets to our page.
+
+To build the UI:
+
+<iframe src="https://demo.arcade.software/5CNFKTzhvnHPrLyZNzgZ?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="500"></iframe>
+
+:::info
+    You can learn more about creating layouts [here](#).
+:::
+
+## 3. Customize Style {#customize-style}
+
+The next step is to customize the style of UI elements. This includes changing the colors, fonts, and sizes of your buttons and labels. In FlutterFlow, you can do this via the [properties panel](#) which provides a range of options for customization.
+
+To customize the style:
+
+<iframe src="https://demo.arcade.software/MGpg8TSzMGBusCGyOk89?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="600"></iframe>
+
+## 4. Add State {#add-state}
+
+Once your UI is set up, it's time to make your app interactive by adding a state. This means setting up your app to respond to user interactions. For example, when a user clicks the button to increase the quantity, the number displayed on the label should increase accordingly.
+
+We can achieve this behavior by adding state variables. A state variable stores data that can change over time. For this specific use case, let's add a [page state variable](#) that will hold the current quantity value. When a user interacts with the buttons, we update this variable, which in turn updates the UI.
+
+Here's how to add and use state variables:
```

Comment:
The section mentions about adding only state variables but Arcade also shows how to add Actions. 
Actions can be a separate step and shouldnt be part of the same Arcade because it makes it very long. 
So please create a new section for adding actions and explanations around it.

Reply:
Ya sure! will do that👍

Reply:
Hey @PoojaB26 I have divided this into subsections now.